docs/api/vortex.development.core.pipelines.md

```diff
@@ -262,10 +263,16 @@ def run(
 **Arguments**:
 
 - `images` _Union[List[str],np.ndarray]_ - list of images path or array of image
+- `coordinate` _str, optional_ - output coordinate format, especially usefull for models that returns
```

Comment:
i think this args need to be named explicitly, rather than only 'coordinates' how about 'output_coordinates_type`

Reply:
how about `'output_coordinate_format'`?

Reply:
done